RedBookPlayer/Player.cs

```diff
@@ -102,12 +103,12 @@ private set
 
                 if (Image != null)
                 {
-                    if (CurrentTrack < Image.Tracks.Count - 1 && CurrentSector >= Image.Tracks[CurrentTrack + 1].TrackStartSector ||
-                        CurrentTrack > 0 && CurrentSector < Image.Tracks[CurrentTrack].TrackStartSector)
+                    if ((CurrentTrack < Image.Tracks.Count - 1 && CurrentSector >= Image.Tracks[CurrentTrack + 1].TrackStartSector)
+                        || (CurrentTrack > 0 && CurrentSector < Image.Tracks[CurrentTrack].TrackStartSector))
                     {
-                        foreach (Track track in Image.Tracks)
+                        foreach (Track track in Image.Tracks.ToArray().Reverse())
```

Comment:
Just reverse should work in a `foreach`

Reply:
Because `Image.Tracks` is `List<Track>`, without the `.ToArray` call, it results in the following error: `CS1579	foreach statement cannot operate on variables of type 'void' because 'void' does not contain a public instance or extension definition for 'GetEnumerator'`.